frank-flow/src/frontend/src/app/shared/models/flow-structure.model.ts

```diff
@@ -73,4 +76,13 @@ export class FlowStructure {
       ) ?? []
     );
   }
+
+  checkImplicitExit(): void {
+    if (!this.nodes.some((node) => node.type === 'Exit')) {
+      this.implicitExit = true;
+      this.lastPipe = this.nodes
+        ?.reverse()
+        .find((node) => node.type.endsWith('Pipe'))?.uid;
```

Comment:
There is a `this.pipes` from which you can take the last one. 

You could just `this.pipes[this.pipes.length - 1]`.

Or do you think this would be better for future expansion?

Reply:
Ill change it to the pipes you suggested.